README.md

```diff
@@ -3,33 +3,73 @@
 This is the home of the web user-interface for Galasa.
 
 ## Technologies used
+
 - [Typescript](https://www.typescriptlang.org/)
 - [React](https://react.dev/)
+- [Next](https://nextjs.org)
 - [Carbon controls](https://carbondesignsystem.com/all-about-carbon/what-is-carbon/)
 
 
-## Set up
-- Clone the repository
-- Run the `setup-locally.sh` script
+When setting up the galasa webui locally, you can either use [(1) our development container (recommended set up)](#1-recommended-set-up-dev-container) or [(2) a manual set up](#2-manual-set-up). Please ensure you've read the [Important Information](#important-information) first.
+
+## Important Information
+
+When attempting to run Galasa's webui locally, you may need to set some environment variables. See below:
+
+### GALASA_DEV_TOKEN and GALASA_API_SERVER_URL
+- In order to [connect the local development web UI with a remote API server](#connecting-the-local-development-web-ui-with-a-remote-api-server), you must set the following environment variables (see link for instructions).
+
+- `GALASA_DEV_TOKEN`: a token that the web UI uses in development mode to connect to a remote service.
+
+- `GALASA_API_SERVER_URL`: the URL of your running Galasa service, e.g. `https://my-galasa-service.com/api`.
+
+### NODE_EXTRA_CA_CERTS and NODE_USE_SYSTEM_CA
+
+- You are required to validate yourself when connecting to a Galasa service. This can be done with `NODE_EXTRA_CA_CERTS` or `NODE_USE_SYSTEM_CA` (only available to the manual approach).
 
-## To build locally
+- `NODE_EXTRA_CA_CERTS`: The path to a PEM file which should contain your intermediate and root CA certificates, e.g. `/Users/user_name/Galasa/certs.pem`. 
 
-To build the code, it needs to get the REST interface definition from somewhere, to generate stubs in order to make calling the REST API easy.
-Hence, we need to tell the build where to get this definition from. 
+- `NODE_USE_SYSTEM_CA` (only available to the manual approach): This tells the UI to ignore certificate errors and trust server certificates if they are in the local system keychain, e.g. `NODE_USE_SYSTEM_CA=1`. This is OK for development purposes, but it is not advised to use this technique on a production deployment of a UI.
```

Comment:
The NODE_USE_SYSTEM_CA env variable doesn't ignore certificate errors, so it's worth removing that part